Makefile

```
@@ -1,3 +1,5 @@
+.PHONY : doc, run, test, coverage, help
```

Only doc needs to be PHONY
```suggestion
.PHONY : doc
```